src/component/elements/NumberInput.tsx

```diff
@@ -29,13 +37,22 @@ const NumberInput = forwardRef(
       step = 'any',
       min = 'any',
       max = 'any',
+      enableAutoSelect = false,
```

Comment:
`autoSelect` seems enough and more simple

Reply:
I wonder if we should have instead `selectOnFocus`, which would be more intuitive than selecting when the component is rendered. It's useless anyway to select the contents if the element is not focused.

Reply:
well, it seems that `.select()` also takes the focus, which is not obvious by reading the MDN doc/examples. So we can keep `autoSelect` for simplicity.